core/src/main/java/com/kin/ecosystem/core/network/model/Error.java

```diff
@@ -65,7 +65,7 @@ public Error message(String message) {
      * @return message
      **/
     public String getMessage() {
-        return message;
+        return message == null ? "" : message;
```

Comment:
change ""  to CONST something like " Missing error message" in case we get Error event without message we will know the origin of the missing message

Reply:
you can also just provide a default value for the property

Reply:
i don't want to use default value because it can be set to null.
i'll use const